db/migrate/20230705213846_update_provider_payment_methods_for_stripe_customers.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class UpdateProviderPaymentMethodsForStripeCustomers < ActiveRecord::Migration[7.0]
+  def change
+    reversible do |dir|
+      dir.up do
+        execute(<<~SQL.squish)
+          UPDATE payment_provider_customers
+          SET settings['provider_payment_methods'] = '["card"]'
```

Comment:
We had issue lately with this syntax for assigning jsonb fields with PG version prior than 14.

The fix was to use `jsonb_set` instead. See https://github.com/getlago/lago-api/pull/1149/files#diff-0ca873c90f6f5b6ae6c7a8a239de5dbd4a79a40e23e43fccb356ae229dadd437R18

Reply:
Thanks @vincent-pochet I changed it to `jsonb_set`.